lib/global-styles.php

```diff
@@ -189,16 +195,6 @@ function gutenberg_experimental_global_styles_resolver( $global_styles ) {
 	return $css_rule;
 }
 
-/**
```

Comment:
Removing this check means that, if the FSE experiment is enabled, we'll load the CSS variables in all wp-admin, not only in edit-site pages.

I did this because the CSS vars are now inlined into the block's CSS, so if I load the edit-post I'd expect it to work the same as the edit-site (variables are loaded, so blocks look the same).

Another option would be to only load this in the edit-post & edit-site? I don't think that's necessary but happy to hear other thoughts. cc @jorgefilipecosta

Reply:
It's worth noting that if #21102 is merged, the editor content will be iframed, so CSS variables outside of that iframe would have no effect.

Reply:
>  so CSS variables outside of that iframe would have no effect.

Something to be mindful of in the iframe PR. We have ways to communicate into and out of the iframe, so it should be possible to set things properly in both.